### PR TITLE
BUGZ-1456/HMD-audio-setting-shows-up-when-desktop-mode

### PR DESCRIPTION
**Pull request by MiladNazeri**
https://highfidelity.atlassian.net/browse/BUGZ-1456

### CODE REVIEW

File: interface/resources/qml/hifi/audio/Audio.qml [hunk 1]
```diff
@@ -266,6 +266,7 @@ Rectangle {
                     labelTextSize: 16;
                     backgroundOnColor: "#E3E3E3";
                     checked: AudioScriptingInterface.warnWhenMuted;
+                    visible: bar.currentIndex === 0 ? false : true;
```
Comment (zfox23) [inline interface/resources/qml/hifi/audio/Audio.qml]:
```suggestion
                    visible: bar.currentIndex !== 0;
```

File: interface/resources/qml/hifi/audio/Audio.qml [hunk 2]
```diff
@@ -277,8 +278,8 @@ Rectangle {
                     id: audioLevelSwitch
                     height: root.switchHeight;
                     switchWidth: root.switchWidth;
-                    anchors.top: warnMutedSwitch.bottom
-                    anchors.topMargin: 24
+                    anchors.top: bar.currentIndex === 0 ? parent.top : warnMutedSwitch.bottom
```
Comment (zfox23) [inline interface/resources/qml/hifi/audio/Audio.qml]:
```suggestion
                    anchors.top: warnMutedSwitch.visible ? warnMutedSwitch.bottom : parent.top
```

File: interface/resources/qml/hifi/audio/Audio.qml [hunk 3]
```diff
@@ -277,8 +278,8 @@ Rectangle {
...
+                    anchors.topMargin: bar.currentIndex === 0 ? 0 : 24
```
Comment (zfox23) [inline interface/resources/qml/hifi/audio/Audio.qml]: where does the number 24 come from?

Comment (MiladNazeri) [reply interface/resources/qml/hifi/audio/Audio.qml]: It looks like all of the top margins for the setting has that

File: interface/resources/qml/hifi/audio/Audio.qml (hunk 2 above)
Comment (MiladNazeri) [reply interface/resources/qml/hifi/audio/Audio.qml]: nice tu

File: interface/resources/qml/hifi/audio/Audio.qml (hunk 1 above)
Comment (MiladNazeri) [reply interface/resources/qml/hifi/audio/Audio.qml]: tu

File: interface/resources/qml/hifi/audio/Audio.qml (hunk 3 above)
Comment (zfox23) [reply interface/resources/qml/hifi/audio/Audio.qml]: so then why wouldn't this be `anchors.topMargin: 24` ?

Comment (MiladNazeri) [reply interface/resources/qml/hifi/audio/Audio.qml]: Let me double-check, when I had that earlier, it wasn't in the same location as the previous HMD setting we took off.

Comment (MiladNazeri) [reply interface/resources/qml/hifi/audio/Audio.qml]: image
with current code
image
with just anchors.topMargin: 24

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21258/HighFidelity-Beta-PR16176-a9da5b8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21258/HighFidelity-Quest-Beta-PR16176-a9da5b8.apk)

File: interface/resources/qml/hifi/audio/Audio.qml (hunk 3 above)
Comment (MiladNazeri) [reply interface/resources/qml/hifi/audio/Audio.qml]: With anchors.topMargin: 24
image

With current Code:
image

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21258/HighFidelity-Beta-PR16176-a9da5b8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21258/HighFidelity-Beta-Interface-PR16176-a9da5b8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21258/HighFidelity-Beta-PR16176-a9da5b8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21258/HighFidelity-Beta-Interface-PR16176-a9da5b8.exe

**Pull request merged**